### DISCUSSION TITLE
Is it possible to use this library in postman pre-request script ?

### DISCUSSION THREAD

**Discussion by anujkumar1012** [panva/jose#276] [Q&A]
Is it possible to use this library in postman pre-request script ? Postman requires, CDN equivalent of the library to load it in an environment variable.

**Comment by panva** [comment]
You can use a CDN like `jsdelivr.net` to load the browser dist files from github tagged releases directly. To do that you may use both the `import` statement as well as the `import()` dynamic ESM loading function in your scripts.

E.g. using a dynamic import

```js
const [
  { jwtVerify }, 
  { createRemoteJWKSet }
] = await Promise.all([
  import('https://cdn.jsdelivr.net/gh/panva/jose@3.19.0/dist/browser/jwt/verify.js'),
  import('https://cdn.jsdelivr.net/gh/panva/jose@3.19.0/dist/browser/jwks/remote.js')
])

const token = 'eyJraWQiOiJNRlplRzEwMmRRaXFiQU5vYU1sV19KbWY3Zk9abXRSc0h0NzdKRmhUcEYwIiwiYWxnIjoiRVMyNTYifQ.eyJmb28iOiJiYXIiLCJpYXQiOjE2MzM0MzU4MDF9.aYumnM7eTWpwIit6WGxFg8bYR3giAdZGzbbPi_4bUnS-HDjk01D50GQXI_A7Bu1pVrw3ELucwmV_vaQ4_v_EUQ'

const JWKS = createRemoteJWKSet(new URL('https://op.panva.cz/jwks'))

const { payload, protectedHeader } = await jwtVerify(token, JWKS)

console.log({ protectedHeader, payload })
```

**Reply by anujkumar1012** [reply]
Thanks for your response. Unfortunately postman doesn't support aync code (there are some workarounds but I am unable to figure out how this particular one can be used). If we can get something like [this](http://kjur.github.io/jsrsasign/jsrsasign-latest-all-min.js) then it will be easier to load it in global variable in one go and then we can easily use the functions in pre-request or test scripts.
